command/unpin.go

```diff
@@ -48,6 +50,7 @@ func (c *UnpinCommand) Flags() *flag.FlagSet {
 	}
 
 	f.StringVar(&c.flagOut, "out", "", "output path (defaults to input file)")
+	f.BoolVar(&c.flagExperimentalFormatNewlines, "experimental-format-newlines", false, "whether or not to undo removing newlines")
```

Comment:
My gut feeling here is that we should do an envvar instead of a flag. That way users can set this in their shell, like `RATCHET_EXP_FORMAT_NEWLINES=true`.

Reply:
Done